### ISSUE TITLE
Exception in async function disable text wrapping

### ISSUE THREAD

**Issue by jcubic** [jcubic/jquery.terminal#998]
### Issue summary

When you have an exception in an async function, the exception doesn't wrap. Also, the default greetings is not responsive anymore. And the exception is throwing on resize there is a warning from jQuery Deferred.

### Expected behavior
Exception in async function should happen only once and the function should be replaced by exception.

### Steps to reproduce
1. Open [repro](https://codepen.io/jcubic/pen/RNbpxbp?editors=0010)
2. Resize the terminal and look into dev tools

### Browser and OS
N/A
